scylla/src/transport/connection.rs

```diff
@@ -1617,6 +1617,10 @@ pub(crate) async fn open_named_connection(
             );
         } else {
             // Fall back to no compression
+            tracing::warn!(
+                "Requested compression <{}> is not supported by the cluster. Falling back to no compression",
+                compression_str
+            );
```

Comment:
when can this happen?

Reply:
This may happen if SUPPORTED message sent by a node does not contain the particular compression string in the value for the `"COMPRESSION"` key.